src/test/java/ch/ivyteam/ivy/maven/engine/download/TestLatestMinorVersionRange.java

```diff
@@ -0,0 +1,15 @@
+package ch.ivyteam.ivy.maven.engine.download;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
```

Comment:
junit4

Reply:
I think <artifactId>maven-plugin-testing-harness</artifactId> is highly built upon junit4 rules. 
And most test are built upon that. 
Maybe there are better ways to write such tests, simulating a maven-plugin and its repo today.